tests/foreman/smoke/test_ui_smoke.py

```diff
@@ -46,7 +46,7 @@ class SmokeTestCase(UITestCase):
     """End-to-end tests using the ``WebUI``."""
 
     def test_positive_find_default_org(self):
-        """@Test: Check if :data:`robottelo.constants.DEFAULT_ORG` is present
```

Comment:
Yes I knew that this change will stop syntax highlight in documentation. I can check and see if I can change testimony as currently it uses `:` for parsing the line as per https://github.com/SatelliteQE/testimony/blob/master/testimony/__init__.py#L118-L121.